tools/f5map/lib/f5map/parser.rb

```diff
@@ -0,0 +1,34 @@
+# Copyright 2018 Google LLC. This software is provided as-is,
+# without warranty or representation for any use or purpose.
+# Your use of it is subject to your agreements with Google.
+
+require 'treetop'
+require 'f5map/node_extensions'
+module F5map
+  ##
+  # The f5 configuration parser.
+  class Parser
+    # Load the grammar
+    Treetop.load(File.expand_path('../f5.treetop', __FILE__))
+    @@parser = F5PersistParser.new
```

Comment:
`@@parser` does not appear to be used in any context other than `.parse` as `.parser is never called`. This should be inlined to `.parse` and removed from the class variables.

Reply:
Ack